azkaban-common/build.gradle

```diff
@@ -25,24 +20,18 @@ model {
 
 dependencies {
   compile('com.google.guava:guava:13.0.1')
-  compile('com.h2database:h2:1.3.170')
```

Comment:
I commented out all the dependencies and enabled them 1 by 1. removed the rest which didn't cause compilation errors.